nullaway/src/test/java/com/uber/nullaway/NullAwayTest.java

```diff
@@ -1233,6 +1233,166 @@ public void OptionalEmptinessHandlerTest() {
         .doTest();
   }
 
+  @Test
+  public void OptionalEmptinessHandlerWithSingleCustomPathTest() {
+    compilationHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:UnannotatedSubPackages=com.uber.lib.unannotated",
+                "-XepOpt:NullAway:CheckOptionalEmptiness=true",
+                "-XepOpt:NullAway:CheckOptionalEmptinessCustomClasses=com.google.common.base.Optional"))
```

Comment:
Please add one more (smaller) test, that checks that `java.util.Optional` is still recognized when this argument is set, as per our discussion. Tests are a good way to document that sort of design edge-case, after all.

Reply:
Done already 👍

Reply:
(True, I missed this one, my bad 😅 )